playground/README.md

```diff
@@ -0,0 +1,68 @@
+# SignIn Widget Playground
+
+## The playground App
```

Comment:
## The playground App

In a nutshell, the playground application demonstrates a customer hosted sign-in widget use-case, which loads sign-in widget assets and bootstrap widget via configurations.

The configuration consists of

- [widgetrc](https://github.com/okta/okta-signin-widget#the-widgetrc-config-file)
- `./main.js`

`main.js` initializes the sign-in widget using settings from `widgetrc` along with a simple success handler.